packages/web3.js-plugin-fee-estimation/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Optimism
```

Comment:
```suggestion
Copyright (c) 2023 Optimism
```

Reply:
Copied from [fee-estimation](https://github.com/ethereum-optimism/optimism/blob/4ef1634c71b55b4e821750272e3ad49de8eba3d7/packages/fee-estimation/LICENSE#L3) package, do we need to update there as well?

Reply:
yea copy pasta. Funny because @nickbalestra left the same comment I think in that pr too but a force push or bad rebase likely ruined it

Reply:
No need to fix it in this pr though it's fine tbh